### PR TITLE
Typos fixed

### PR DESCRIPTION
**Pull request by isralduke**
This PR fixes typos across 32 files.

### CODE REVIEW

File: docs/accessibility/skip-link.md [hunk 1]
```diff
@@ -13,7 +13,8 @@ eleventyNavigation:
 
 ## Best Practice
 
-- Users who us assistive technology users or those who use keyboard navigation need a way to skip past navigation elements if they are already familiar with the page.
+Users who rely keyboard navigation need a way to skip past navigation elements if they are already familiar with the page.
```
Comment (frogpad7) [inline docs/accessibility/skip-link.md:16]:
```suggestion
Users who rely on keyboard navigation need a way to skip past navigation elements if they are already familiar with the page.
```

File: docs/components/pagination.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ eleventyNavigation:
 - Use Pagination if data, such as on [Tables](/components/table), is long and needs to be split up.
 - Make sure the Pagination clearly shows the current page’s number.
 - Pagination is placed at the bottom of the content which it pages.
-- You may need to adjust columns sizes for best fir to your project.
+- You may need to adjust columns sizes for best fit to your project.
```
Comment (frogpad7) [inline docs/components/pagination.md:19]:
```suggestion
- You may need to adjust columns sizes to best fit to your project.
```

File: docs/components/skip-link.md [hunk 3]
```diff
@@ -13,7 +13,7 @@ eleventyNavigation:
 
 ## Best Practice
 
-Assistive technology users or those who use keyboard navigation need a way to skip past navigation elements if they are already familiar with the page.
+Users who rely keyboard navigation need a way to skip past navigation elements if they are already familiar with the page.
```
Comment (frogpad7) [inline docs/components/skip-link.md:16]:
```suggestion
Users who rely on keyboard navigation need a way to skip past navigation elements if they are already familiar with the page.
```

File: docs/feature-blocks/call-to-action-double.md [hunk 4]
```diff
@@ -15,7 +15,7 @@ eleventyNavigation:
 
 Use this when you’re ready to direct the reader to take an action, but that action must be a decision among two presented. 
   - Do not present more than two choices.
-  - This Call-to-Action, like the Call-to-Action Single, is not navigation, it iss a persuasion with two choices you present to the user.
+  - This Call-to-Action, like the Call-to-Action Single, is not navigation, it’s a persuasion item with two choices you present to the user.
```
Comment (frogpad7) [inline docs/feature-blocks/call-to-action-double.md:18]:
```suggestion
  - This Call-to-Action, like the Call-to-Action Single, is not for navigation; it’s a persuasion item with two choices you present to the user.
```

File: docs/feature-blocks/call-to-action-single.md [hunk 5]
```diff
@@ -15,7 +15,7 @@ eleventyNavigation:
 
 Use this when you’re ready to direct the reader to take an action, like signing up.
   - The big text is the action verb for the goal you have for the user. Use this to direct the user an action. This is a decision moment, don’t put a lot of text in here.
-  - A Call-to-Action is not navigation, it is a persuasion with a link to what you want the user to do.
+  - A Call-to-Action is not navigation, it’s a persuasion item with a link to what you want the user to do.
```
Comment (frogpad7) [inline docs/feature-blocks/call-to-action-single.md:18]:
```suggestion
  - A Call-to-Action is not navigation. It’s a persuasion item with a link to what you want the user to do.
```

**Review by frogpad7 — CHANGES_REQUESTED** [review]
Just a few more grammar adjustments to note; the links work perfectly fine.

**Review by frogpad7 — DISMISSED** [review]
All current text changes are much more readable! Nice cleanup!

**Review by rltucker — COMMENTED** [review]
1. I would argue 'Cards' should actually be 'Card' since other areas use the singular to refer to an element.
2. Why is the 'required' note removed in Radio? I've actually had devs ask how to add it. We can't assume they will know to go to 'Input' to see it. It really should be mentioned as many times as possible.
3. Images: We should keep the 250 character suggestion. 'Short but descriptive is vague'

**Review by rltucker — APPROVED** [review]
Approved

**Pull request merged**